nx.json

```diff
@@ -21,6 +21,9 @@
     "todo": { "tags": [] },
     "todo-data-access": { "tags": [] },
     "tasks": { "tags": [] },
```

Comment:
"task" doesn't tell us what it is.

Reply:
jasne, to jest biblioteka nadrzędna do feature, data-access i ui, jak to najlepiej byłoby nazwać?

Reply:
nie potrzebujesz tego przetrzymywać jako bibliotekę. Możesz zostawić to jako zwykły folder.